README.md

```diff
@@ -3,28 +3,25 @@
 
 Rancher Dashboard is the UI that powers [Rancher](https://www.rancher.com/products/rancher).
 
-Developer documentation and documentation for our UI components is available here - [Rancher UI Devkit](https://rancher.github.io/dashboard/).
-
-Rancher Dashboard supports an extension mechanism that allows developers to independently provide additional functionality to Rancher. You can learn more from our [Rancher Extensions Docs](https://rancher.github.io/dashboard/extensions/introduction).
-
+Rancher Dashboard supports an extension mechanism that allows developers to independently provide additional functionality to Rancher. You can learn more from our [Rancher Extensions Docs](https://extensions.rancher.io).
 
 # What is it?
 
 Rancher Dashboard provides a sophisticated UI for managing Kubernetes clusters and Workloads.
 
 <img src="docusaurus/docs/getting-started/screenshots/home.png" />
 
-For more information, see our [UI Walk-through documentation](https://rancher.github.io/dashboard/getting-started/ui-walkthrough).
+## Internal Developer Docs
+
+These docs are intended only for Dashboard UI developers.
 
-## Developer Docs
+If you're building extensions, please see: https://extensions.rancher.io.
 
-- [Rancher UI Devkit](https://rancher.github.io/dashboard/) - for our documentation home
-- [Developer Quick Start documentation](https://rancher.github.io/dashboard/getting-started/quickstart) - to get the Rancher Dashboard running
```

Comment:
nit pick - might be useful to either keep a direct link to how to run the dashboard within the README (new URL is https://extensions.rancher.io/internal/getting-started/quickstart) or a clear comment the getting started guide is in the docs